### PR TITLE
feat(jackett) add jackett to apps

### PR DESCRIPTION
**Pull request by xstar97**
**Description**
Custom jackett app.
⚒️ Fixes  # 

**⚙️ Type of change**

- [ ] ⚙️ Feature/App addition
- [ ] 🪛 Bugfix
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🔃 Refactor of current code

**🧪 How Has This Been Tested?**

**📃 Notes:**
round 2

**✔️ Checklist:**

- [ ] ⚖️ My code follows the style guidelines of this project
- [ ] 👀 I have performed a self-review of my own code
- [ ] #️⃣ I have commented my code, particularly in hard-to-understand areas
- [ ] 📄 I have made corresponding changes to the documentation
- [ ] ⚠️ My changes generate no new warnings
- [ ] 🧪 I have added tests to this description that prove my fix is effective or that my feature works
- [ ] ⬆️ I increased versions for any altered app according to semantic versioning

**➕ App addition**

If this PR is an app addition please make sure you have done the following.

- [ ] 🪞 I have opened a PR on [truecharts/containers](https://github.com/truecharts/containers) adding the container to TrueCharts mirror repo.
- [ ] 🖼️ I have added an icon in the Chart's root directory called `icon.png`

---

_Please don't blindly check all the boxes. Read them and only check those that apply.
Those checkboxes are there for the reviewer to see what is this all about and
the status of this PR with a quick glance._

### CODE REVIEW

**Comment by xstar97** [comment]
i think it passed more test than it did last time.

File: apps/jackett/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+#shellcheck disable=SC2086
+exec \
+    /app/jackett \
+        --NoUpdates \
+        "$@"
```
Comment (stavros-k) [inline apps/jackett/entrypoint.sh:7]: Is there a flag for the config directory? I remember most of those *arr apps have one

File: apps/jackett/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+LABEL "org.opencontainers.image.source"="https://github.com/truecharts/apps"
+
+ARG CONTAINER_NAME
+ARG CONTAINER_VER
+LABEL org.opencontainers.image.title="${CONTAINER_NAME}"
+LABEL org.opencontainers.image.url="https://truecharts.org/docs/charts/${CONTAINER_NAME}"
+LABEL org.opencontainers.image.version="${CONTAINER_VER}"
+LABEL org.opencontainers.image.description="Container for ${CONTAINER_NAME} by TrueCharts"
+LABEL org.opencontainers.image.documentation="https://truecharts.org/docs/charts/${CONTAINER_NAME}"
```
Comment (stavros-k) [inline apps/jackett/Dockerfile:50]:
```suggestion
LABEL org.opencontainers.image.documentation="https://truecharts.org/docs/charts/${CONTAINER_NAME}"

```

File: apps/jackett/goss.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+# https://github.com/aelsabbahy/goss/blob/master/docs/manual.md#port
+port:
+  # https://github.com/aelsabbahy/goss/issues/149
+  tcp6:9117:
+    listening: true
+
+# https://github.com/aelsabbahy/goss/blob/master/docs/manual.md#http
+http:
+  http://localhost:9117/UI/Login:
+    status: 200
```
Comment (stavros-k) [inline apps/jackett/goss.yaml:11]:
```suggestion
    status: 200

```

File: apps/jackett/entrypoint.sh (hunk 1 above)
Comment (xstar97) [reply apps/jackett/entrypoint.sh:7]: not sure

Comment (stavros-k) [reply apps/jackett/entrypoint.sh:7]: https://github.com/Jackett/Jackett/blob/9f365c5651a7f46f2d4a64b68013f5e78bb2ea8a/README.md?plain=1#L890

Maybe add `--PORT ${JACKETT_PORT}`

File: apps/jackett/entrypoint.sh [hunk 4]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+#shellcheck disable=SC2086
+exec \
+    /app/jackett \
+        --NoUpdates \
+        "$@" \
+        --PORT \
```
Comment (stavros-k) [inline apps/jackett/entrypoint.sh:8]:
```suggestion
        --Port \
```

**Pull request merged**